lib/storages/global-parameter.storage.ts

```diff
@@ -0,0 +1,22 @@
+import { ParameterObject } from '../interfaces/open-api-spec.interface';
+
+export class GlobalParameterStorageHost {
+  private parameters = new Array<ParameterObject>();
+
+  addGlobalParameters(...parameter: ParameterObject[]) {
+    this.parameters.push(...parameter);
+  }
+
+  getGlobalParameters() {
+    return this.parameters;
+  }
+
+  cleanGlobalParamters() {
+    this.parameters = [];
+  }
+}
```

Comment:
```suggestion
export class GlobalParametersStorageHost {
  private parameters = new Array<ParameterObject>();

  add(...parameter: ParameterObject[]) {
    this.parameters.push(...parameter);
  }

  getAll() {
    return this.parameters;
  }

  clear() {
    this.parameters = [];
  }
}
```

Reply:
Thank you for taking the time to review my commits! I have changed some names as you suggested.